.golangci.yml

```diff
@@ -376,6 +376,10 @@ linters-settings:
     forbid:
       # Builtin function:
       - ^print(ln)?$
+      - {p: "^(.*)log\\.Error$", msg: "ErrorContext must be called"}
```

Comment:
- Add comment with section header, e.g. "# slogx.CtxHandler support:".
- Maybe make sense to rephrase like "Use ErrorContext to support slogx.CtxHandler."

Move these rules into separate project, e.g.:
- `testdata/lint/.golangci.yml`
- `testdata/lint/go.mod`
- `testdata/lint/main.go`